creator/lib/screens/home_screen.dart

```diff
@@ -56,4 +81,186 @@ class _HomeScreenState extends State<HomeScreen> {
+          children: [
+            Ink.image(
+              image: NetworkImage(
+                'https://images.unsplash.com/photo-1514888286974-6c03e2ca1dba?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1327&q=80',
+              ),
+              child: InkWell(
+                onTap: () {},
+              ),
+              height: 240,
+              fit: BoxFit.cover,
+            ),
+            Text(
+              titleList[index],
+              style: TextStyle(
+                  fontWeight: FontWeight.bold,
+                  color: Colors.white,
+                  fontSize: 24),
+            ),
+          ],
+        ),
+      );
+}
+
+class CarouselWithDotsPage extends StatefulWidget {
+  List<String> imgList;
+  CarouselWithDotsPage({this.imgList, Key key}) : super(key: key);
+
+  @override
+  _CarouselWithDotsPageState createState() => _CarouselWithDotsPageState();
+}
+
+class _CarouselWithDotsPageState extends State<CarouselWithDotsPage> {
+  int _current = 0;
+
+  Widget _bottomTextOfCarousel(String item) {
+    return Positioned(
+      bottom: 0.0,
+      left: 0.0,
+      right: 0.0,
+      child: Container(
+        decoration: BoxDecoration(
+          gradient: LinearGradient(
+            colors: [
+              Color.fromARGB(200, 0, 0, 0),
+              Color.fromARGB(0, 0, 0, 0),
+            ],
+            begin: Alignment.bottomCenter,
+            end: Alignment.topCenter,
+          ),
+        ),
+        padding: EdgeInsets.symmetric(
+          horizontal: 20,
+          vertical: 10,
+        ),
+        child: Text(
+          "No. ${widget.imgList.indexOf(item)} image",
+          style: TextStyle(
+            color: Colors.white,
+            fontSize: 12.0,
+            fontWeight: FontWeight.bold,
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _dotPageStateOfCarousel() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: widget.imgList.map((url) {
+        int index = widget.imgList.indexOf(url);
+        return Container(
+          width: 8,
+          height: 8,
+          margin: EdgeInsets.symmetric(
+            vertical: 10,
+            horizontal: 3,
+          ),
+          decoration: BoxDecoration(
+            shape: BoxShape.circle,
+            color: _current == index
+                ? Color.fromRGBO(0, 0, 0, 0.9)
+                : Color.fromRGBO(0, 0, 0, 0.4),
+          ),
+        );
+      }).toList(),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final List<Widget> imageSliders = widget.imgList
+        .map((item) => Container(
+            child: ClipRRect(
+                borderRadius: BorderRadius.all(Radius.circular(20.0)),
+                child: Stack(
+                  children: [
+                    Image.network(item, fit: BoxFit.cover, width: 1000),
+                    _bottomTextOfCarousel(item),
+                  ],
+                ))))
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-a-trailing-comma-for-arguments-parameters-and-list-items-but-only-if-they-each-have-their-own-line

Reply:
I changed

Reply:
I fixed it.